task/zap-baseline/0.1/zap-baseline.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: zap-baseline
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: cli
+    tekton.dev/displayName: "OWASP Zap Baseline"
+spec:
+  description: >-
+    OWASP Zap Baseline is a command line tool,
+    testing api endpoints for vulnerabilities.
+
+  workspaces:
+    - name: zap-folder
+      description: A RW folder where all zap output files is stored
+      mountPath: /zap/wrk
+  params:
+    - name: testURL
+      description: URL of to be tested, must include https:// or http://
+      type: string
+      default: ""
+    - name: configFile
+      description: Path to zap-scan-config
+      type: string
+      default: scan.conf
+    - name: cmName
+      description: Name of cm containing scan.conf
+      type: string
+      default: zap-scan-config
+    - name: mins
+      description: The number of minutes to spider for
+      type: string
+      default: "1"
+  steps:
+    - name: zap-baseline-scan
+      image: quay.io/nissessenap/zap-proxy:0.0.1
```

Comment:
This seems to be your personal repository which is fine... but is the project planning to offer their own image?

Reply:
See: https://github.com/tektoncd/catalog/pull/455#issuecomment-666437894